aiohttp_sentry/__init__.py

```diff
@@ -1,22 +1,49 @@
+import asyncio
 from functools import partial
+import sys
 
 import raven
 import raven_aiohttp
 
 
 class SentryMiddleware:
 
-    def __init__(self, sentry_kwargs=None):
+    def __init__(self, install_excepthook=True, sentry_kwargs=None):
         if sentry_kwargs is None:
             sentry_kwargs = {}
 
+        if install_excepthook:
+            # do not let raven.Client install its own excepthook
+            sentry_kwargs['install_sys_hook'] = False
```

Comment:
This is much simpler if you just add `'install_sys_hook': not install_excepthook` below in the `sentry_kwargs` dict.